From: Ian Thomas <ianthomas23@gmail.com>
Subject: [PATCH 4/4] Add missing matio.cmake.in

cmake/matio.cmake.in is used by cmake/src.cmake but is missing from the
1.6.0 release tarball. Contents taken from the upstream v1.6.0 git tag.
---
 cmake/matio.cmake.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cmake/matio.cmake.in b/cmake/matio.cmake.in
new file mode 100644
--- /dev/null
+++ b/cmake/matio.cmake.in
@@ -0,0 +1,3 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@MATIO_EXPORT_NAME@.cmake")
